src/modal/modal.scss

```diff
@@ -153,3 +128,67 @@
     opacity: 1;
   }
 }
+
+@mixin iui-modal-title {
+  font-size: inherit;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  line-height: 1.5;
+}
+
+@mixin iui-modal-title-bar {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: $iui-baseline $iui-m;
+  box-sizing: border-box;
+  font-size: $iui-font-size-subheading;
+
+  @at-root .iui-modal-resizable & {
+    user-select: none;
+    font-size: $iui-font-size-leading;
+    padding: round($iui-baseline * 0.5) $iui-m;
+    cursor: grab;
+    @include themed {
+      background-color: t(iui-color-background-3);
```

Comment:
Is background color of the title bar in our standards? I'm not sure I like that simple and draggable/resizable modals have different title bars.

Reply:
Jon did the visual work on this; I'll ping him to respond also. I know that the resizable modal's title bar is a different color to make it apparent that it's draggable (and resizable, I suppose?). If they look the same how can we signal to users that some are draggable/resizable?

Reply:
I was trying to match [what is currently implemented in AppUI](https://github.com/iTwin/iTwinUI-react/issues/614).  I think we need a bar area for the user to grab.  If you have any visual design ideas of how you'd like it to look, I'm open to investigating alternative designs.

Reply:
Darker bar area does not necessarily means that item is draggable.
We have this drag handle icon which could be included in title: https://itwin.github.io/iTwinUI-icons/#icons-drag-handle and I tried to think where (if) we could include it in bar area and it just doesn't fit.
So probably it's best to have darker title bar for draggable dialogs/modals. Especially when people are already using it like this.